Source/WebCore/html/HTMLOptionElement.idl

```diff
@@ -31,6 +31,6 @@
     attribute boolean selected;
     [CEReactions=NotNeeded] attribute [AtomString] DOMString value;
 
-    [CEReactions=NotNeeded, CEReactions] attribute DOMString text;
+    [CEReactions] attribute DOMString text;
```

Comment:
This still modifies text??

Reply:
@rniwa - I slept and didn't see this. I didn't get your comment, with this change of only keeping `CEReactions`, we are passing previously failing WPT test.

Reply:
@rniwa - Should I merge this?